### PR TITLE
Improvement: Google Map block not able to select in editor after drag drop 

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
When we drag and drop the block in the editor and then we are not able to select google map again to change its settings
issue --> https://share.getcloudapp.com/7Kup9Ogr

[Trello](https://trello.com/c/ec4jPoo8/212-fix-google-map-block-not-able-to-select-again-to-change-settings)

### Types of changes
- Added padding to the main div in the editor 

### How has this been tested?
Tested whether we are able to select block easily in editor.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 ### 1.22.0 ###
 * Fix: Post Layouts - Fixed the JS undefined error with `post_type` in block editor while using Post Masonry.
+* Improvement: Google Map - Now user can easily select block in editor.
```
Comment (vrundakansara) [inline README.md:172]: `* Fix: Google Map - Added some padding for the editor to improve the experience of a selection of Google Map.`

File: src/blocks/google-map/style.scss [hunk 2]
```diff
@@ -5,8 +5,8 @@
  */
 
 .uagb-google-map__wrap {
-
 	display: flex;
+	padding: 25px;
```
Comment (vrundakansara) [inline src/blocks/google-map/style.scss:9]: Why are we adding padding in the frontend?
Will this affect existing designs?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (supriyajjadhav) [reply README.md]: Updated readme Added 1.22.2 version

File: src/blocks/google-map/style.scss (hunk 2 above)
Comment (supriyajjadhav) [reply src/blocks/google-map/style.scss:8]: Updated CSS. Added padding to the block editor only.

Comment (vrundakansara) [reply src/blocks/google-map/style.scss:8]: Will this CSS be loaded on frontend too?

Comment (supriyajjadhav) [reply src/blocks/google-map/style.scss]: Yes.. Added editor.scss file to load this CSS only on editor side

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**